index.html

```diff
@@ -503,6 +519,7 @@
                                 priorities.appendChild(box);
                             }
                         }
+                        lowestPriority = priorityName;
```

Comment:
Maybe we should be resetting to default priority, can we determine that somehow?